deploy/28_factory_deployed_contracts.ts

```diff
@@ -65,9 +65,50 @@ const func: DeployFunction = async function (hre1: HardhatRuntimeEnvironment) {
       });
       hre.deployments.log('Deployed a "CxipERC721Proxy" empty contract for block explorer verification purposes.');
     }
+
+    const holographUtilityToken: Contract | null = await hre.ethers.getContractOrNull(
+      'HolographUtilityToken',
+      deployer
+    );
+    if (holographUtilityToken == null) {
+      await hre.deployments.deploy('HolographUtilityToken', {
+        ...(await txParams({
+          hre,
+          from: deployer,
+          to: '0x0000000000000000000000000000000000000000',
+          gasLimit: await hre.ethers.provider.estimateGas(
+            (await hre.ethers.getContractFactory('HolographUtilityToken')).getDeployTransaction()
+          ),
+        })),
+        args: [],
+        log: true,
+        waitConfirmations: 1,
+      });
+      hre.deployments.log(
+        'Deployed a "HolographUtilityToken" empty contract for block explorer verification purposes.'
+      );
+    }
+
+    const hToken: Contract | null = await hre.ethers.getContractOrNull('hToken', deployer);
+    if (hToken == null) {
+      await hre.deployments.deploy('hToken', {
+        ...(await txParams({
+          hre,
+          from: deployer,
+          to: '0x0000000000000000000000000000000000000000',
+          gasLimit: await hre.ethers.provider.estimateGas(
+            (await hre.ethers.getContractFactory('hToken')).getDeployTransaction()
+          ),
+        })),
+        args: [],
+        log: true,
+        waitConfirmations: 1,
+      });
+      hre.deployments.log('Deployed a "hToken" empty contract for block explorer verification purposes.');
+    }
   }
 };
 
 export default func;
-func.tags = ['Holographer4verify', 'CxipERC721Proxy4verify'];
+func.tags = ['Holographer4verify', 'CxipERC721Proxy4verify', 'HolographUtilityToken4verify', 'hToken4verify'];
```

Comment:
What is this `hToken4verify`?

Reply:
Just a tag name to use in case we want to skip rest of deploy scripts and just run deployment for specific contracts that need to be verified.

Reply:
I see